config/cmake/hdf5-config.cmake.in

```diff
@@ -92,7 +92,7 @@ if (${HDF5_PACKAGE_NAME}_ENABLE_PARALLEL)
   find_package(MPI QUIET REQUIRED)
 endif ()
 
-if (${HDF5_PACKAGE_NAME}_ENABLE_THREADSAFE OR ${HDF5_PACKAGE_NAME}_ENABLE_SUBFILING_VFD)
+if (${HDF5_PACKAGE_NAME}_ENABLE_THREADSAFE OR ${HDF5_PACKAGE_NAME}_ENABLE_THREADS OR ${HDF5_PACKAGE_NAME}_ENABLE_SUBFILING_VFD)
```

Comment:
The "${HDF5_PACKAGE_NAME}_ENABLE_THREADSAFE" and "${HDF5_PACKAGE_NAME}_ENABLE_SUBFILING_VFD)" components are no longer correct and should be removed.

(Leaving just ${HDF5_PACKAGE_NAME}_ENABLE_THREADS)

Reply:
Why are they no longer correct, if they are set in the library? This file is the status of the options used to build the library.

Reply:
Do you mean that since they require threads that is why we only need to check threads?

Reply:
This check is only about threads, not threadsafety or subfiling.